src/org/joml/AABBd.java

```diff
@@ -283,6 +283,55 @@ public double getMin(int component) throws IllegalArgumentException {
         }
     }
 
+    /**
+     * Return the length along the x component
+     *
+     * @return length in the x dimension
+     */
+    public double lengthX(){
+        return maxX - minX;
+    }
+
+    /**
+     * Return the length along the y component
+     *
+     * @return length in the y dimension
+     */
+    public double lengthY(){
+        return maxY - minY;
+    }
+
+    /**
+     * Return the length along the z component
+     *
+     * @return length in the z dimension
+     */
+    public double lengthZ(){
+        return maxZ - minZ;
+    }
+
+    /**
+     * Get the size of the aabb.
+     *
+     * @param dest
+     *         will hold the result
+     * @return dest
+     */
+    public Vector3f getSize(Vector3f dest) {
+        return dest.set(maxX - minX, maxY - minY, maxZ - minZ);
```

Comment:
Is this because of performance concerns, or why not use `lengthX` etc? 
```suggestion
        return dest.set(lengthX(), lengthY(), lengthZ());
```